apps/condo/domains/ticket/access/TicketAnalyticsReportService.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Generated by `createservice ticket.TicketAnalyticsReportService`
+ */
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+async function canReadTicketAnalyticsReport ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.isAdmin) return true
+    return { organization: { employees_some: { user: { id: user.id, isBlocked: false } } } }
```

Comment:
Please, write tests to check access to another organization's data!

SECURITY ISSUE!

https://v5.keystonejs.com/api/access-control#custom-schema-access-control

Reply:
Added test for this case & fix access